webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingBucketHelper.scala

```diff
@@ -82,9 +83,22 @@ trait VolumeTracingBucketHelper
       .flatten
   }
 
-  def saveBucket(dataLayer: VolumeTracingLayer, bucket: BucketPosition, data: Array[Byte], version: Long): Fox[Unit] = {
+  def loadBucketFromCache(dataLayer: VolumeTracingLayer, bucket: BucketPosition) = {}
+
+  def saveBucket(dataLayer: VolumeTracingLayer,
+                 bucket: BucketPosition,
+                 data: Array[Byte],
+                 version: Long,
+                 toCache: Boolean = false): Fox[Unit] = {
     val key = buildBucketKey(dataLayer.name, bucket)
-    volumeDataStore.put(key, version, compressVolumeBucket(data, expectedUncompressedBucketSizeFor(dataLayer)))
+    val compressedBucket = compressVolumeBucket(data, expectedUncompressedBucketSizeFor(dataLayer))
+    if (toCache) {
+      // Note that this cache is for temporary volumes only (e.g. compound projects)
+      // and cannot be used for download or versioning
+      Fox.successful(volumeDataCache.insert(key, compressedBucket, Some(20 minutes)))
```

Comment:
I think this shouldn't be hardcoded to 20 minutes. I don't know if we need to expose this as a config settings, but there might be different preferences based on the RAM, etc.